src/ar/mangaswat/src/eu/kanade/tachiyomi/extension/ar/mangaswat/MangaSwat.kt

```diff
@@ -23,11 +23,11 @@ import java.util.Locale
 
 class MangaSwat : MangaThemesia(
     "MangaSwat",
-    "https://goldragon.me",
+    "https://swatmanhua.com",
```

Comment:
create a constant out of the class (top/bottom of this file) for the url and use that here in the constructor and in `defaultBaseUrl `

Reply:
> create a constant out of the class (top/bottom of this file) for the url and use that here in the constructor and in `defaultBaseUrl `

done